proto/exocore/operator/v1/genesis.proto

```diff
@@ -12,10 +12,129 @@ option go_package = "github.com/ExocoreNetwork/exocore/x/operator/types";
 message GenesisState {
   // there are no params for this module.
   // operators is a list of the registered operators.
-  repeated OperatorInfo operators = 1 [(gogoproto.nullable) = false];
-  // TODO: add non-chainID AVS opt-in information for exporting / importing.
+  repeated OperatorDetail operators = 1 [(gogoproto.nullable) = false];
+
+  // add other information for exporting / importing.
   // Although it is not necessary for the bootstrapped genesis, it is
   // necessary for chain restarts.
+  // operator_records refers to a list of operator records. each record
+  // contains an operator address and a list of chain id +
+  // cons key combination.
+  repeated OperatorConsKeyRecord operator_records = 2
```

Comment:
I plan to remove the opt-in from the InitGenesis of Dogfood and retain the changes from #PR108 that set the consensus key and opted information in the operator module. This will ensure that the imported states remain consistent with the pre-export states—for example, the opted height will be preserved, and the USD value of operators won’t be reset to zero by the opt-in during genesis initialization. This needs to be tested to see if it works well.